Gemfile

```diff
@@ -46,3 +46,5 @@ group :test do
   gem "simplecov", require: false
   gem "vcr", require: false
 end
+
+gem "rtl"
```

Comment:
Can this be moved to the list of gems at the top of the file?

Reply:
Sure, yeah.